docs/guides/operator-guide-azure-enclave.md

```diff
@@ -9,129 +9,310 @@ sidebar_position: 18
 
 ```
-cat uid2-operator-azure-sgx.tar.gz | docker import - uid20.azurecr.io/uid2/operator/occlum:0.2
-docker push uid20.azurecr.io/uid2/operator/occlum:0.2
+az group create --name {RESOURCE_GROUP_NAME} --location {LOCATION}
 ```
 
-3. Craft the k8s deployment and service as required. You can use `uid2-operator-aks.yml` as an example.
+:::info
+All the resources are provisioned later under the name you provide as the {RESOURCE_GROUP_NAME} value.
+:::
 
->IMPORTANT: Make sure that the operator image targets SGX enabled nodes (for example, by specifying a resource limit
-for `kubernetes.azure.com/sgx_epc_mem_in_MiB`).
+There are some limitations with regard to location:
+- UID2 Private Operator for Azure is not supported in these areas: Europe, China.
 
-Operator service is designed to run on Standard_DC8_v2 instances. Running on smaller nodes is not supported.
-Running multiple operator service instances on same node is not supported.
+- For Azure virtual network deployment availability, check [Linux container groups](https://learn.microsoft.com/en-us/azure/container-instances/container-instances-region-availability#linux-container-groups) in the Azure documentation. You can only deploy to regions with the **Confidential SKU** column set to **Y** in the table.
 
-### Advanced Deployment Options
+### Complete Key Vault and Managed Identity Setup
 
->IMPORTANT: These are not recommended practices, but they may be useful for testing purposes.
+The next step is to set up a [key vault](https://learn.microsoft.com/en-us/azure/key-vault/general/overview) and save the operator key in it.
 
-#### Docker
+When you've created the key vault, you can create a [managed identity](https://learn.microsoft.com/en-us/entra/identity/managed-identities-azure-resources/overview) and grant it permission to access the key vault.
 
-Make sure the host has Intel SGX DCAP driver installed and configured. Follow the steps from the AKS section
-to make the operator docker image available for running.
+Later [Azure Container Instances](https://azure.microsoft.com/en-us/products/container-instances) (ACIs) will launch as this identity.
 
-Example of running the image:
+Follow these steps:
 
-```
-docker run \
-        --device /dev/sgx/enclave --device /dev/sgx/provision \
-        -p 8080:8080 \
-        -p 9091:9091 \
-        ghcr.io/iabtechlab/uid2-operator-azure-occlum:dev
-```
+1. Update the `vault.parameters.json` file with the following required values:
 
-#### Direct Invocation of Occlum (Advanced)
+   | Parameter | Description |
+   | :--- | :--- |
+   | `vaultName` | The name of the key vault for hosting the operator key secret. The name you choose must be globally unique. |
+   | `operatorKeyValue` | The `OPERATOR_KEY` secret value, which you received from the UID team as part of account setup (see [Set Up UID2 Operator Account](#set-up-uid2-operator-account)). This value is unique to you, like a password: keep it secure and secret. |
```

Comment:
Passwords are not necessarily unique. Did you mean to say something like `This value is unique to you and acts like a password`

Reply:
Updated. Thx.